src/components/navbar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Is this needed?

Reply:
good point, will fix this now

Reply:
we dont need React for this component becuase it is not being used